### ISSUE TITLE
Yum update to latest daily build fails...

### ISSUE THREAD

**Issue by sathnaga** [open-power-host-os/builds#313]
<cde:info> Mirrored with LTC bug https://bugzilla.linux.ibm.com/show_bug.cgi?id=168294 </cde:info>

```
# yum clean all
Loaded plugins: fastestmirror, product-id, search-disabled-repos, subscription-manager
This system is not registered with an entitlement server. You can use subscription-manager to register.
Cleaning repos: base epel extras hostos updates
Cleaning up everything
Maybe you want: rm -rf /var/cache/yum, to also free up space taken by orphaned data from disabled or removed repos
Cleaning up list of fastest mirrors

# yum update
Loaded plugins: fastestmirror, product-id, search-disabled-repos, subscription-manager
This system is not registered with an entitlement server. You can use subscription-manager to register.
Determining fastest mirrors
epel/ppc64le/metalink                                                                                                                                    |  16 kB  00:00:00     
 * epel: mirror.math.princeton.edu
base                                                                                                                                                     | 3.6 kB  00:00:00     
epel                                                                                                                                                     | 4.7 kB  00:00:00     
extras                                                                                                                                                   | 2.9 kB  00:00:00     
hostos                                                                                                                                                   | 2.9 kB  00:00:00     
updates                                                                                                                                                  | 2.9 kB  00:00:00     
(1/8): base/7/ppc64le/group_gz                                                                                                                           | 166 kB  00:00:00     
(2/8): epel/ppc64le/group_gz                                                                                                                             |  88 kB  00:00:00     
(3/8): base/7/ppc64le/primary_db                                                                                                                         | 4.7 MB  00:00:00     
(4/8): epel/ppc64le/updateinfo                                                                                                                           | 928 kB  00:00:00     
(5/8): extras/7/ppc64le/primary_db                                                                                                                       | 107 kB  00:00:00     
(6/8): hostos/primary_db                                                                                                                                 | 649 kB  00:00:00     
(7/8): updates/7/ppc64le/primary_db                                                                                                                      | 1.7 MB  00:00:00     
(8/8): epel/ppc64le/primary_db                                                                                                                           | 5.9 MB  00:00:00     
Resolving Dependencies
--> Running transaction check
---> Package kernel.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be installed
---> Package kernel-bootwrapper.ppc64le 0:4.16.0-4.dev.gitfd8742e.el7.centos.p will be updated
---> Package kernel-bootwrapper.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be an update
---> Package kernel-debug-devel.ppc64le 0:4.16.0-4.dev.gitfd8742e.el7.centos.p will be updated
---> Package kernel-debug-devel.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be an update
---> Package kernel-debuginfo.ppc64le 0:4.16.0-4.dev.gitfd8742e.el7.centos.p will be updated
---> Package kernel-debuginfo.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be an update
---> Package kernel-debuginfo-common-ppc64le.ppc64le 0:4.16.0-4.dev.gitfd8742e.el7.centos.p will be updated
---> Package kernel-debuginfo-common-ppc64le.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be an update
---> Package kernel-devel.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be installed
---> Package kernel-headers.ppc64le 0:4.16.0-4.dev.gitfd8742e.el7.centos.p will be updated
---> Package kernel-headers.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be an update
---> Package kernel-tools.ppc64le 0:4.16.0-4.dev.gitfd8742e.el7.centos.p will be updated
---> Package kernel-tools.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be an update
---> Package kernel-tools-libs.ppc64le 0:4.16.0-4.dev.gitfd8742e.el7.centos.p will be updated
---> Package kernel-tools-libs.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be an update
---> Package open-power-host-os-all.noarch 0:3.5-36.dev.el7.centos.p will be updated
---> Package open-power-host-os-all.noarch 0:3.5-37.dev.el7 will be an update
--> Processing Dependency: servicelog = 1.1.14-9.dev.git15f2af5.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: qemu = 15:2.12.0-2.dev.gitd36f3ee.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: libvpd = 2.2.5-8.dev.git7d959c5.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: libvirt = 4.3.0-1.dev.git3096ff1.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: libservicelog = 1.1.18-3.dev.git1e39e77.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: hwdata = 0.288-3.dev.git625a119.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: golang-github-shurcooL-sanitized_anchor_name = 1:0-3.dev.git1dba4b3.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: golang-github-russross-blackfriday = 1:1.2-8.dev.git5f33e7b.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: SLOF = 20171214-2.dev.gitc2a331f.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
---> Package open-power-host-os-base.noarch 0:3.5-36.dev.el7.centos.p will be updated
---> Package open-power-host-os-base.noarch 0:3.5-37.dev.el7 will be an update
---> Package open-power-host-os-container.noarch 0:3.5-36.dev.el7.centos.p will be updated
---> Package open-power-host-os-container.noarch 0:3.5-37.dev.el7 will be an update
--> Processing Dependency: kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7 for package: open-power-host-os-container-3.5-37.dev.el7.noarch
---> Package open-power-host-os-ras.noarch 0:3.5-36.dev.el7.centos.p will be updated
---> Package open-power-host-os-ras.noarch 0:3.5-37.dev.el7 will be an update
--> Processing Dependency: servicelog = 1.1.14-9.dev.git15f2af5.el7 for package: open-power-host-os-ras-3.5-37.dev.el7.noarch
--> Processing Dependency: libvpd = 2.2.5-8.dev.git7d959c5.el7 for package: open-power-host-os-ras-3.5-37.dev.el7.noarch
--> Processing Dependency: libservicelog = 1.1.18-3.dev.git1e39e77.el7 for package: open-power-host-os-ras-3.5-37.dev.el7.noarch
--> Processing Dependency: hwdata = 0.288-3.dev.git625a119.el7 for package: open-power-host-os-ras-3.5-37.dev.el7.noarch
---> Package open-power-host-os-release.noarch 0:3.5-36.dev.el7.centos.p will be updated
---> Package open-power-host-os-release.noarch 0:3.5-37.dev.el7 will be an update
---> Package open-power-host-os-virt.noarch 0:3.5-36.dev.el7.centos.p will be updated
---> Package open-power-host-os-virt.noarch 0:3.5-37.dev.el7 will be an update
--> Processing Dependency: qemu = 15:2.12.0-2.dev.gitd36f3ee.el7 for package: open-power-host-os-virt-3.5-37.dev.el7.noarch
--> Processing Dependency: libvirt = 4.3.0-1.dev.git3096ff1.el7 for package: open-power-host-os-virt-3.5-37.dev.el7.noarch
--> Processing Dependency: SLOF = 20171214-2.dev.gitc2a331f.el7 for package: open-power-host-os-virt-3.5-37.dev.el7.noarch
---> Package perf.ppc64le 0:4.16.0-4.dev.gitfd8742e.el7.centos.p will be updated
---> Package perf.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be an update
---> Package procps-ng.ppc64le 0:3.3.10-17.el7 will be updated
---> Package procps-ng.ppc64le 0:3.3.10-17.el7_5.2 will be an update
---> Package python-perf.ppc64le 0:4.16.0-4.dev.gitfd8742e.el7.centos.p will be updated
---> Package python-perf.ppc64le 0:4.16.0-5.dev.git5ed4ce3.el7 will be an update
--> Finished Dependency Resolution
--> Running transaction check
---> Package kernel-devel.ppc64le 0:4.15.0-5.dev.git33f711f.el7.centos will be erased
---> Package open-power-host-os-all.noarch 0:3.5-37.dev.el7 will be an update
--> Processing Dependency: servicelog = 1.1.14-9.dev.git15f2af5.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: qemu = 15:2.12.0-2.dev.gitd36f3ee.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: libvpd = 2.2.5-8.dev.git7d959c5.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: libvirt = 4.3.0-1.dev.git3096ff1.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: libservicelog = 1.1.18-3.dev.git1e39e77.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: hwdata = 0.288-3.dev.git625a119.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: golang-github-shurcooL-sanitized_anchor_name = 1:0-3.dev.git1dba4b3.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: golang-github-russross-blackfriday = 1:1.2-8.dev.git5f33e7b.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
--> Processing Dependency: SLOF = 20171214-2.dev.gitc2a331f.el7 for package: open-power-host-os-all-3.5-37.dev.el7.noarch
---> Package open-power-host-os-container.noarch 0:3.5-37.dev.el7 will be an update
--> Processing Dependency: kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7 for package: open-power-host-os-container-3.5-37.dev.el7.noarch
---> Package open-power-host-os-ras.noarch 0:3.5-37.dev.el7 will be an update
--> Processing Dependency: servicelog = 1.1.14-9.dev.git15f2af5.el7 for package: open-power-host-os-ras-3.5-37.dev.el7.noarch
--> Processing Dependency: libvpd = 2.2.5-8.dev.git7d959c5.el7 for package: open-power-host-os-ras-3.5-37.dev.el7.noarch
--> Processing Dependency: libservicelog = 1.1.18-3.dev.git1e39e77.el7 for package: open-power-host-os-ras-3.5-37.dev.el7.noarch
--> Processing Dependency: hwdata = 0.288-3.dev.git625a119.el7 for package: open-power-host-os-ras-3.5-37.dev.el7.noarch
---> Package open-power-host-os-virt.noarch 0:3.5-37.dev.el7 will be an update
--> Processing Dependency: qemu = 15:2.12.0-2.dev.gitd36f3ee.el7 for package: open-power-host-os-virt-3.5-37.dev.el7.noarch
--> Processing Dependency: libvirt = 4.3.0-1.dev.git3096ff1.el7 for package: open-power-host-os-virt-3.5-37.dev.el7.noarch
--> Processing Dependency: SLOF = 20171214-2.dev.gitc2a331f.el7 for package: open-power-host-os-virt-3.5-37.dev.el7.noarch
--> Finished Dependency Resolution
Error: Package: open-power-host-os-ras-3.5-37.dev.el7.noarch (hostos)
           Requires: libvpd = 2.2.5-8.dev.git7d959c5.el7
           Installed: libvpd-2.2.5-8.dev.git7d959c5.el7.centos.p.ppc64le (@hostos)
               libvpd = 2.2.5-8.dev.git7d959c5.el7.centos.p
           Available: libvpd-2.2.5-1.el7.ppc64le (base)
               libvpd = 2.2.5-1.el7
           Available: libvpd-2.2.5-8.dev.git7d959c5.el7.ppc64le (hostos)
               libvpd = 2.2.5-8.dev.git7d959c5.el7
Error: Package: open-power-host-os-ras-3.5-37.dev.el7.noarch (hostos)
           Requires: servicelog = 1.1.14-9.dev.git15f2af5.el7
           Installed: servicelog-1.1.14-9.dev.git15f2af5.el7.centos.p.ppc64le (@hostos)
               servicelog = 1.1.14-9.dev.git15f2af5.el7.centos.p
           Available: servicelog-1.1.14-3.el7.ppc64le (base)
               servicelog = 1.1.14-3.el7
           Available: servicelog-1.1.14-9.dev.git15f2af5.el7.ppc64le (hostos)
               servicelog = 1.1.14-9.dev.git15f2af5.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: libvpd = 2.2.5-8.dev.git7d959c5.el7
           Installed: libvpd-2.2.5-8.dev.git7d959c5.el7.centos.p.ppc64le (@hostos)
               libvpd = 2.2.5-8.dev.git7d959c5.el7.centos.p
           Available: libvpd-2.2.5-1.el7.ppc64le (base)
               libvpd = 2.2.5-1.el7
           Available: libvpd-2.2.5-8.dev.git7d959c5.el7.ppc64le (hostos)
               libvpd = 2.2.5-8.dev.git7d959c5.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: hwdata = 0.288-3.dev.git625a119.el7
           Installed: hwdata-0.288-3.dev.git625a119.el7.centos.p.ppc64le (@hostos)
               hwdata = 0.288-3.dev.git625a119.el7.centos.p
           Available: hwdata-0.252-8.8.el7.ppc64le (base)
               hwdata = 0.252-8.8.el7
           Available: hwdata-0.288-3.dev.git625a119.el7.ppc64le (hostos)
               hwdata = 0.288-3.dev.git625a119.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: servicelog = 1.1.14-9.dev.git15f2af5.el7
           Installed: servicelog-1.1.14-9.dev.git15f2af5.el7.centos.p.ppc64le (@hostos)
               servicelog = 1.1.14-9.dev.git15f2af5.el7.centos.p
           Available: servicelog-1.1.14-3.el7.ppc64le (base)
               servicelog = 1.1.14-3.el7
           Available: servicelog-1.1.14-9.dev.git15f2af5.el7.ppc64le (hostos)
               servicelog = 1.1.14-9.dev.git15f2af5.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: libvirt = 4.3.0-1.dev.git3096ff1.el7
           Installed: libvirt-4.3.0-1.dev.git3096ff1.el7.centos.p.ppc64le (@hostos)
               libvirt = 4.3.0-1.dev.git3096ff1.el7.centos.p
           Available: libvirt-3.9.0-14.el7.ppc64le (base)
               libvirt = 3.9.0-14.el7
           Available: libvirt-3.9.0-14.el7_5.2.ppc64le (updates)
               libvirt = 3.9.0-14.el7_5.2
           Available: libvirt-3.9.0-14.el7_5.4.ppc64le (updates)
               libvirt = 3.9.0-14.el7_5.4
           Available: libvirt-3.9.0-14.el7_5.5.ppc64le (updates)
               libvirt = 3.9.0-14.el7_5.5
           Available: libvirt-4.3.0-1.dev.git3096ff1.el7.ppc64le (hostos)
               libvirt = 4.3.0-1.dev.git3096ff1.el7
Error: Package: open-power-host-os-virt-3.5-37.dev.el7.noarch (hostos)
           Requires: SLOF = 20171214-2.dev.gitc2a331f.el7
           Installed: SLOF-20171214-2.dev.gitc2a331f.el7.centos.p.noarch (@hostos)
               SLOF = 20171214-2.dev.gitc2a331f.el7.centos.p
           Available: SLOF-20170724-2.git89f519f.el7.noarch (base)
               SLOF = 20170724-2.git89f519f.el7
           Available: SLOF-20171214-2.dev.gitc2a331f.el7.noarch (hostos)
               SLOF = 20171214-2.dev.gitc2a331f.el7
Error: Package: open-power-host-os-virt-3.5-37.dev.el7.noarch (hostos)
           Requires: qemu = 15:2.12.0-2.dev.gitd36f3ee.el7
           Installed: 15:qemu-2.12.0-2.dev.gitd36f3ee.el7.centos.p.ppc64le (@hostos)
               qemu = 15:2.12.0-2.dev.gitd36f3ee.el7.centos.p
           Available: 15:qemu-2.12.0-2.dev.gitd36f3ee.el7.ppc64le (hostos)
               qemu = 15:2.12.0-2.dev.gitd36f3ee.el7
Error: Package: open-power-host-os-virt-3.5-37.dev.el7.noarch (hostos)
           Requires: libvirt = 4.3.0-1.dev.git3096ff1.el7
           Installed: libvirt-4.3.0-1.dev.git3096ff1.el7.centos.p.ppc64le (@hostos)
               libvirt = 4.3.0-1.dev.git3096ff1.el7.centos.p
           Available: libvirt-3.9.0-14.el7.ppc64le (base)
               libvirt = 3.9.0-14.el7
           Available: libvirt-3.9.0-14.el7_5.2.ppc64le (updates)
               libvirt = 3.9.0-14.el7_5.2
           Available: libvirt-3.9.0-14.el7_5.4.ppc64le (updates)
               libvirt = 3.9.0-14.el7_5.4
           Available: libvirt-3.9.0-14.el7_5.5.ppc64le (updates)
               libvirt = 3.9.0-14.el7_5.5
           Available: libvirt-4.3.0-1.dev.git3096ff1.el7.ppc64le (hostos)
               libvirt = 4.3.0-1.dev.git3096ff1.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7
           Installed: kubernetes-1.2.0-0.23.dev.git4a3f9c5.el7.centos.p.ppc64le (@hostos)
               kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7.centos.p
           Available: kubernetes-1.2.0-0.23.dev.git4a3f9c5.el7.ppc64le (hostos)
               kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: qemu = 15:2.12.0-2.dev.gitd36f3ee.el7
           Installed: 15:qemu-2.12.0-2.dev.gitd36f3ee.el7.centos.p.ppc64le (@hostos)
               qemu = 15:2.12.0-2.dev.gitd36f3ee.el7.centos.p
           Available: 15:qemu-2.12.0-2.dev.gitd36f3ee.el7.ppc64le (hostos)
               qemu = 15:2.12.0-2.dev.gitd36f3ee.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: libservicelog = 1.1.18-3.dev.git1e39e77.el7
           Installed: libservicelog-1.1.18-3.dev.git1e39e77.el7.centos.p.ppc64le (@hostos)
               libservicelog = 1.1.18-3.dev.git1e39e77.el7.centos.p
           Available: libservicelog-1.1.18-1.el7.ppc64le (base)
               libservicelog = 1.1.18-1.el7
           Available: libservicelog-1.1.18-3.dev.git1e39e77.el7.ppc64le (hostos)
               libservicelog = 1.1.18-3.dev.git1e39e77.el7
Error: Package: open-power-host-os-ras-3.5-37.dev.el7.noarch (hostos)
           Requires: libservicelog = 1.1.18-3.dev.git1e39e77.el7
           Installed: libservicelog-1.1.18-3.dev.git1e39e77.el7.centos.p.ppc64le (@hostos)
               libservicelog = 1.1.18-3.dev.git1e39e77.el7.centos.p
           Available: libservicelog-1.1.18-1.el7.ppc64le (base)
               libservicelog = 1.1.18-1.el7
           Available: libservicelog-1.1.18-3.dev.git1e39e77.el7.ppc64le (hostos)
               libservicelog = 1.1.18-3.dev.git1e39e77.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: golang-github-russross-blackfriday = 1:1.2-8.dev.git5f33e7b.el7
           Installed: 1:golang-github-russross-blackfriday-1.2-8.dev.git5f33e7b.el7.centos.p.noarch (@hostos)
               golang-github-russross-blackfriday = 1:1.2-8.dev.git5f33e7b.el7.centos.p
           Available: 1:golang-github-russross-blackfriday-1.2-8.dev.git5f33e7b.el7.noarch (hostos)
               golang-github-russross-blackfriday = 1:1.2-8.dev.git5f33e7b.el7
Error: Package: open-power-host-os-container-3.5-37.dev.el7.noarch (hostos)
           Requires: kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7
           Installed: kubernetes-1.2.0-0.23.dev.git4a3f9c5.el7.centos.p.ppc64le (@hostos)
               kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7.centos.p
           Available: kubernetes-1.2.0-0.23.dev.git4a3f9c5.el7.ppc64le (hostos)
               kubernetes = 1.2.0-0.23.dev.git4a3f9c5.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: SLOF = 20171214-2.dev.gitc2a331f.el7
           Installed: SLOF-20171214-2.dev.gitc2a331f.el7.centos.p.noarch (@hostos)
               SLOF = 20171214-2.dev.gitc2a331f.el7.centos.p
           Available: SLOF-20170724-2.git89f519f.el7.noarch (base)
               SLOF = 20170724-2.git89f519f.el7
           Available: SLOF-20171214-2.dev.gitc2a331f.el7.noarch (hostos)
               SLOF = 20171214-2.dev.gitc2a331f.el7
Error: Package: open-power-host-os-ras-3.5-37.dev.el7.noarch (hostos)
           Requires: hwdata = 0.288-3.dev.git625a119.el7
           Installed: hwdata-0.288-3.dev.git625a119.el7.centos.p.ppc64le (@hostos)
               hwdata = 0.288-3.dev.git625a119.el7.centos.p
           Available: hwdata-0.252-8.8.el7.ppc64le (base)
               hwdata = 0.252-8.8.el7
           Available: hwdata-0.288-3.dev.git625a119.el7.ppc64le (hostos)
               hwdata = 0.288-3.dev.git625a119.el7
Error: Package: open-power-host-os-all-3.5-37.dev.el7.noarch (hostos)
           Requires: golang-github-shurcooL-sanitized_anchor_name = 1:0-3.dev.git1dba4b3.el7
           Installed: 1:golang-github-shurcooL-sanitized_anchor_name-0-3.dev.git1dba4b3.el7.centos.p.noarch (@hostos)
               golang-github-shurcooL-sanitized_anchor_name = 1:0-3.dev.git1dba4b3.el7.centos.p
           Available: 1:golang-github-shurcooL-sanitized_anchor_name-0-3.dev.git1dba4b3.el7.noarch (hostos)
               golang-github-shurcooL-sanitized_anchor_name = 1:0-3.dev.git1dba4b3.el7
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

**Comment by fabianorosas** [comment]
@sathnaga Let me give you a bit of background on this. When CentOS released 7.5, there was an issue in their centos-release package which contained the /etc/rpm/macros.dist file with the `%{dist}` value set to `el7.centos.p` instead of the expected `el7` or `el7.centos` (from 7.4). Our builds merely picked that up and not knowing how CentOS would decide to fix that, we decided not to apply a fix on our side and wait for the official solution.

I see that the build nodes now have the updated centos-release package so our builds should start producing `el7` packages. The `-37` you just tried is the first of them.
 
I'm afraid the cleanest solution here is to uninstall the offending packages (*.centos.p) and retry the update.

**Comment by sathnaga** [comment]
@fabianorosas Worked fine with removing (*.centos.p) and update..
